cmd/main.go

```diff
@@ -321,3 +331,16 @@ func (a *ApiEnabler) Enable() {
 	a.isEnabled = true
 	a.log.Info("API is enabled")
 }
+
+func automigrationWithLeader(k8sClient *kubernetes.Clientset, db *gorm.DB, log logrus.FieldLogger) error {
+	migrationLeader := leader.NewElector(k8sClient, leader.Config{LeaseDuration: 5 * time.Second,
+		RetryInterval: 2 * time.Second, Namespace: Options.LeaderConfig.Namespace, RenewDeadline: 4 * time.Second},
+		"assisted-service-migration-helper",
+		log.WithField("pkg", "migrationLeader"))
+	return migrationLeader.RunWithLeader(context.Background(), func() error {
+		log.Infof("Start automigration")
```

Comment:
So in 2 out of 3 replica we will get an error here?

Reply:
why? we will not